src/condor_utils/log_rotate.cpp

```diff
@@ -70,7 +70,7 @@ long long quantizeTimestamp(time_t tt, long long secs)
 {
 	if ( ! secs) return tt;
 
-	static int leap_sec = -1;
+	static time_t leap_sec = -1;
```

Comment:
Is `time_t` signed?

Reply:
Yes, otherwise you could not represent times before the epoch (January 1, 1970).